packages/fxa-shared/README.md

```diff
@@ -62,7 +61,7 @@ implied by another scope:
 
 Or you can find out
 what values in the set
-are *not* implied by another scope:
+are _not_ implied by another scope:
```

Comment:
Interesting that it's going to modify all our docs too. Are we all agreed there's value in that? Is turning it off for `.md` files an option (and would we want to consider it if it was)?

Reply:
I was a bit surprised by this too though have come around to it.